plugins/modules/vault_token_create.py

```diff
@@ -39,10 +40,10 @@
   attributes:
     check_mode:
       support: partial
-      description:
+      details:
         - In check mode, this module will not create a token, and will instead return a basic structure with an empty token.
           However, this may not be useful if the token is required for follow on tasks.
-          It may be better to use this module with I(check_mode=no) in order to have a valid token that can be used.
+        - It may be better to use this module with I(check_mode=no) in order to have a valid token that can be used.
```

Comment:
Same here.
```suggestion
        - It may be better to use this module with I(check_mode=false) in order to have a valid token that can be used.
```